documenten/techniek/designdecisions.md

```diff
@@ -13,3 +13,19 @@ We gebruiken NLX voor elke call waarbij vertrouwelijke gegevens worden opgevraag
 
 We hanteren semantic versioning. 
 We hanteren https://docs.geostandaarden.nl/api/API-Strategie/#versioning  als uitgangspunt voor versiebeheer
+
+
+On standards and standardization
+-------
+The common ground principles are leading in design choices, and within those principles international compliancy and technological invocation is deemed most important. **We do not want to make concessions to the current infrastructure.** As such the component might differ on [NL API Strategie](https://docs.geostandaarden.nl/api/API-Strategie), [NORA](https://www.noraonline.nl/wiki/Standaarden), [zaakgericht werken](https://zaakgerichtwerken.vng.cloud/themas/index) and or other standards if they are deemed incompatible or out of line with (inter)national standards and or good practices. 
+
+Unfortunately (inter)national standards can be conflicting. We therefore prioritize standards on several grounds
+
+- International is put before local
```

Comment:
Lokale standaarden zouden juist een specialisatie moeten zijn van internationaal. Als we willen aansluiten bij landelijke standaarden kunnen we denk ik niet stellen dat internationaal voor landelijk gaat. Eerder:
We maken gebruik van landelijke standaarden. Als er geen landelijke standaard is, maken we indien beschikbaar gebruik van een internationale standaard.

Reply:
Ja daar denk ik echt anders over :) Voor zover de NL standaarden een specialisatie zijn van zullen ze ook niet strijdig zijn. Dus daar zit denk ik geen Issue.

Waar wel een issue zit is dat de NL standaarden zo nu en dan op best wel heftige of gekke dingen strijdig zijn met internationale standaarden. Op die punten dienen we de NL standaarden aan te passen en de benodigde aanpassingen voor de desbetreffende organisaties inzichtelijk te maken. 

Dit zit overigens ook al in de NL standaarden zelf, ongeveer ieder document begint er mee dat ze compliant willen zijn aan de internationale standaarden. Dus daar moeten ze dan wat mee worden geholpen.

Ik denk ook dat het in veel gevallen nog niet eens een bewuste afwijking is, maar gewoon niet weten dat er een standaard voor is. Error berichten is daar volgens mij ook weer een voorbeeld van.